data/ideas.json

```
@@ -238,6 +238,12 @@
         "description": "A web application comprising major days in history including births, deaths, inventions, wars, etc. User can search for days, events or persons using an API.",
         "level": "intermediate",
         "tags": "major-days-in-history, search"
+    },
+    {
+        "title": "Digital_Clock with time formatting",
```

Remove the underscore
```suggestion
        "title": "Digital Clock with time formatting",
```